jujugui/static/gui/src/app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -514,68 +554,59 @@
 
         .header-slot {
             header {
-                padding: $inspector-main-padding;
+                padding: $inspector-main-padding / 2 $inspector-main-padding;
+                border-bottom: 1px solid $inspector-border-color;
 
                 .service-charm {
-                    color: $charm-panel-background-color;
-                    width: 100%;
-
                     .icon {
-                        margin-right: 10px;
-                        vertical-align: top;
+                        float: right;
+                        margin-top: 3px;
                     }
+
                     .icon,
                     .icon img {
                         /* Required for sizing images in IE10 */
-                        width: 50px;
-                        height: 50px;
+                        width: 24px;
+                        height: 24px;
 
                     }
-                    .details-wrapper {
-                        width: 180px;
-                        margin-top: 3px;
+
+                    .service-name {
+                        width: 190px;
+                    }
+
+                    div.service-name {
```

Comment:
There's a distinction between div.service-name and input.service-name for edit/non-edit.